src/tools/contrib/balance-cpu.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python
+"""
+This script gets the output from lscpu and produces a list of CPU uids
+corresponding to physical cores, intended to use to allocate Seastar reactors
+in a balanced way across sockets.
+
+Two strategies of balancing reactors over CPU cores:
+
+1) OSD based: all the reactors of each OSD run in the same CPU NUMA socket (default),
+2) Socket based: reactors for the same OSD are distributed evenly across CPU NUMA sockets.
```

Comment:
Are we aware of any advantages of socket based allocation(2)? Is it there only for comparison purposes?

Reply:
Precisely, for comparison, once the full runs are completed (currently on o05) I should be able to produce a detailed comparison and upload charts 👍